linode_api4/groups/account.py

```diff
@@ -448,3 +449,31 @@ def user_create(self, email, username, restricted=True):
 
         u = User(self.client, result["username"], result)
         return u
+
+    def enrolled_betas(self, *filters):
+        """
+        Returns a list of all Beta Programs an account is enrolled in.
+
+        API doc: TBD
+
+        :returns: a list of Beta Programs.
+        :rtype: PaginatedList of AccountBetaProgram
+        """
+        return self.client._get_and_filter(AccountBetaProgram, *filters)
+
+    def join_beta_program(self, id):
```

Comment:
Optional suggestion: Maybe this method could also accept a [BetaProgramObject](https://github.com/linode/linode_api4-python/pull/321/files#diff-a73d26f66cba4c755411eb73cc9fea45214c64150960860e7ab36a36b37c5307R4) similar to how we accept a Region or region ID for the `instance_create` method: https://github.com/linode/linode_api4-python/blob/dev/linode_api4/groups/linode.py#L289